WhaleSpotting/react-app/src/components/PageNav.tsx

```diff
@@ -1,22 +1,26 @@
-import React, { useState } from "react";
+import React from "react";
 import "../styles/PageNav.scss";
 import { Button, Style } from "./Button";
 
-export default function PageNav(): JSX.Element {
-    const [page, setPage] = useState(1);
+interface pageNavProps {
+    nextPage: () => void;
+    previousPage: () => void;
+}
+
+export default function PageNav({ nextPage, previousPage }: pageNavProps): JSX.Element {
 
     return (
         <div className="page-nav" data-testid="page-nav">
             <Button
```

Comment:
Can we make it so the previous page only shows up when we are on page 2 or more?

Reply:
Done